Demo/UI Tests/American Express UI Tests/AmericanExpress_UITests.swift

```diff
@@ -19,7 +19,7 @@ class AmericanExpress_UITests: XCTestCase {
         app.buttons["Valid card"].tap()
         sleep(2)
 
-        XCTAssertTrue(app.buttons["45256433 Points, 316795.03 USD"].waitForExistence(timeout: 10))
+        XCTAssertTrue(app.buttons["45256433 Points, 316795.03 USD"].waitForExistence(timeout: 20))
```

Comment:
While there are several instances in this PR just bumping timeouts, several of these test were also hitting the same timeouts locally. If the test completes _before_ time timeout is hit it'll move on to the next test so there is no real harm in bumping the timeouts for the recurring failing tests.